### PR TITLE
Add mergify

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

Automate the backport with mergify

## Why is it important?

Simplify the backport process with github labels.

## Actions

- Already enabled by infra for this project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,22 @@ pre-commit installed at .git/hooks/pre-commit
 ```
 
 To understand more about the hooks we use, please take a look at pre-commit's [configuration file](./.pre-commit-config.yml).
+
+## Backports
+
+This project requires to backport to the existing active branches, those branches are defined in the `.backportrc.json` and `.mergify.yml` files. In order to do so,
```
Comment (cachedout) [inline README.md:37]:
```suggestion
This project requires backports to the existing active branches. Those branches are defined in the `.backportrc.json` and `.mergify.yml` files. In order to do so,
```

File: README.md [hunk 2]
```diff
@@ -31,3 +31,22 @@ pre-commit installed at .git/hooks/pre-commit
...
+there are two different approaches:
+
+### Mergify 🥇
+
+This is the preferred approach, backports are created automatically as long as the rules defined in [.mergify.yml](.mergify.yml) are fulfilled. From the user's point of
```
Comment (cachedout) [inline README.md:42]:
```suggestion
This is the preferred approach. Backports are created automatically as long as the rules defined in [.mergify.yml](.mergify.yml) are fulfilled. From the user's point of
```

**Review by cachedout — APPROVED** [review]
👍 Just minor fix-ups.

**Comment by mdelapenya** [comment]
Awesome!

image

**Pull request merged**